lib/collective-sections.js

```diff
@@ -445,6 +445,45 @@ export const convertSectionsToNewFormat = sections => {
   return convertedSections;
 };
 
+/**
+ * Converts sections back to legacy format ([{ section, isEnabled }]), without categories
+ */
+export const convertSectionsToLegacyFormat = sections => {
```

Comment:
Great - so now we can convert both ways? 🎉

Reply:
Exactly :-)

Reply:
Old frontend will work with new format